Modules/Core/Support/JsonErrorHelper.php

```
@@ -0,0 +1,32 @@
+        // Think of a better name for this function. It doesn't return
+        // json itself but is called from something which will.
+        $return = [];
+    
+        foreach (array_keys($_POST) as $key) {
+            if (form_error($key)) {
+                $return[$key] = form_error($key);
+            }
+        }
+    
+        return $return;
+    }
```

PHPMD (2.15.0)

json_errors accesses the super-global variable $_POST. (undefined)

(Superglobals)